muxconfig/main.c

```diff
@@ -112,11 +112,11 @@ void init_navigation_groups() {
     apply_theme_list_panel(&theme, &device, ui_pnlServices);
     apply_theme_list_panel(&theme, &device, ui_pnlRTC);
 
```

Comment:
The _("WordsToTranslate) markup makes it easy to identify what is being translated.  Anything with this markup will call the translate method in common.c.  If a translation exists it will use that but if not it uses what is passed.  This also has the benefit of being able to automatically scrape and build a default json file for distribution